src/main/java/net/minestom/server/particle/ParticleOptionImpl.java

```diff
@@ -0,0 +1,392 @@
+package net.minestom.server.particle;
+
+
+import net.minestom.server.color.Color;
+import net.minestom.server.coordinate.Point;
+import net.minestom.server.entity.Entity;
+import net.minestom.server.instance.block.Block;
+import net.minestom.server.item.ItemStack;
+import org.jetbrains.annotations.NotNull;
+
+import static net.minestom.server.particle.ParticleOption.*;
+
+public interface ParticleOptionImpl {
+
+    static ParticleOption AMBIENT_ENTITY_EFFECT() {
```

Comment:
I'd prefer using camelCase instead of SCREAMING_SNAKE_CASE for these method names